pkg/api/helpers.go

```diff
@@ -52,3 +54,19 @@ func GetFieldLabelConversionFunc(supportedLabels map[string]string, overrideLabe
 		return "", "", fmt.Errorf("field label not supported: %s", label)
 	}
 }
+
+// TODO: Move this upstream
+func ExportObjectMeta(objMeta *kapi.ObjectMeta, exact bool) {
```

Comment:
Generally if `exact` is passed I'd leave the `Name` or `GenerateName` where applicable and `Namespace`. All other fields are not important. At least that's what I usually do when doing oc get -o yaml and modifying by hand.